hermes-consumers/src/main/java/pl/allegro/tech/hermes/consumers/supervisor/process/ConsumerProcess.java

```diff
@@ -63,7 +63,7 @@ public void run() {
         try {
             Thread.currentThread().setName("consumer-" + getSubscriptionName());
 
-            while (running && !Thread.interrupted()) {
+            while (running && !Thread.currentThread().isInterrupted()) {
```

Comment:
if we use Thread.interrupted() this fact will be useless because flag indicating interruption is cleared every time the method is called